src/pages/steps/StepsForm.tsx

```diff
@@ -0,0 +1,174 @@
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Button, ButtonVariants } from '@/ui/Button';
+import { Inline } from '@/ui/Inline';
+import { Link, LinkVariants } from '@/ui/Link';
+import { Page } from '@/ui/Page';
+import { Text } from '@/ui/Text';
+import { getValueFromTheme } from '@/ui/theme';
+import { Toast } from '@/ui/Toast';
+
+import { FormData } from '../manage/movies/MovieForm';
+import { useToast } from '../manage/movies/useToast';
+import { useAddEvent } from './hooks/useAddEvent';
+import { useEditField } from './hooks/useEditField';
+import { FooterStatus, useFooterStatus } from './hooks/useFooterStatus';
+import { useGetEvent } from './hooks/useGetEvent';
+import { useParseStepConfiguration } from './hooks/useParseStepConfiguration';
+import { usePublishEvent } from './hooks/usePublishEvent';
+import { PublishLaterModal } from './modals/PublishLaterModal';
+import { Steps, StepsConfiguration } from './Steps';
+
+const getValue = getValueFromTheme('createPage');
+
+type StepsFormProps = {
+  configuration: StepsConfiguration<FormData>;
+  convertFormDataToEvent: (data: Object) => Object;
+  convertEventToFormData: (event: Object) => Object;
+  toastConfiguration: Object;
+  title: string;
+  label?: string;
+};
+
+const StepsForm = ({
+  configuration,
```

Comment:
```suggestion
  stepConfigurations,
```

I would rename this prop so that it is clear you're passing an array of configurations for the steps